Sources/MapboxNavigation/BannerContainerView.swift

```diff
@@ -1,16 +1,20 @@
 import UIKit
 
 // :nodoc:
-@objc(MBBannerContainerView)
-open class BannerContainerView: UIView {
+public class Banner {
     
     // :nodoc:
-    public enum `Type` {
-        case top
-        case bottom
+    public enum Position {
+        case topLeading
+        case bottomLeading
```

Comment:
Will be potentially extended to support also `topTrailing` and `bottomTrailing` cases, when top and bottom banner containers are shown on the right side of the screen.